data/shwetasng.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "Shweta-Singh",
+  "type": "personal",
+  "bio": "**Sophomore pursuing Btech CSE**",
+
+  "tags": ["DevOps", "WebDev", "Open Source"],
+
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/<Your Twitter Username>" },
+    { "icon": "FaGithub", "url": "https://github.com/shwetasng" },
+    {
+      "icon": "FaLinkedin",
+      "url": "www.linkedin.com/in/shweta-singh-2b4a82225"
+    }
+  ],
+
+  "links": [
+    {
+      "name": "GitHub: Let's collaborate",
+      "url": "https://github.com/shwetasng",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/<Your Twitter Username>",
+      "icon": "FaTwitter"
+    },
```

Add the Twitter username, if you don't have remove that part.
```suggestion
    {
      "name": "Twitter: Follow me",
      "url": "https://twitter.com/<Your Twitter Username>",
      "icon": "FaTwitter"
    },
```